e2e/playwright.config.ts

```diff
@@ -32,6 +32,8 @@ export default defineConfig({
     trace: 'on-first-retry',
   },
 
+  snapshotPathTemplate: '__screenshots__/{testFilePath}/{arg}{ext}',
```

Comment:
is the place where this compare is done?

Reply:
i.e. it will compare the screenshot agains this file?

Reply:
no that the template format to generate the file ( image in this case ) to when there is no file ( on fist run )

Reply:
i tweaked this because by default it include this browser os version name ( darwin-chrome while on ci it would do lunux-chrome ) , this is only needed if you want to do mutil browser testing, which we do not do ( at this moment ) and expect divergences per browser. say for instance you want to regenerate all your snapshot expectation you can add the regenerate flag and you could generate images for browser/os/device combination ( for example firefox on android ) but i would suggest not to go there for now and only use this for stuff like this logo where then is no clear text or dom element to test for

Reply:
it uses https://github.com/mapbox/pixelmatch , which has some really cool ideas ( _pixelmatch is around 150 lines of code, has no dependencies,)_